public/lectures_list.php

```diff
@@ -0,0 +1,108 @@
+<?php require_once(dirname(__FILE__)."/../modules/ensure_logged_in.php"); ?>
+
+<?php
+    require "../modules/models/course.php";
+    require "../modules/models/lecture.php";
+
+    $course_id = $_GET['id'];
+
+    try {
+        $specific_course = Course::where(array('id' => $course_id));
```

Comment:
instead of `where` you could use `find_by`, it'd return the first Course that matches the search condition (it is also nullable)

Reply:
Cool! I'll switch that shortly!